src/main/java/generated/uk/gov/govtalk/people/bs7666/BLPUextentStructure.java

```diff
@@ -0,0 +1,260 @@
+//
+// This file was generated by the Eclipse Implementation of JAXB, v4.0.3
+// See https://eclipse-ee4j.github.io/jaxb-ri
+// Any modifications to this file will be lost upon recompilation of the source schema.
+//
+
+
+package uk.gov.govtalk.people.bs7666;
```

Comment:
is this package name correct ? seems weird to have numbers.

Reply:
its generated, so guess it must be